### PR TITLE
ref(feature-disabled): Use expandable alert UI

### PR DESCRIPTION
**Pull request by vuluongj20**
**Before:**
Screen Shot 2022-03-31 at 2 50 20 PM
Screen Shot 2022-03-31 at 2 50 36 PM

**After:**
Screen Shot 2022-04-05 at 11 18 31 AM
Screen Shot 2022-04-05 at 11 18 39 AM

### CODE REVIEW

File: static/app/components/acl/featureDisabled.tsx [hunk 1]
```diff
@@ -70,30 +70,20 @@ class FeatureDisabled extends React.Component<Props, State> {
     this.setState(state => ({showHelp: !state.showHelp}));
   };
 
-  renderFeatureDisabled() {
+  renderContent() {
```
Comment (evanpurkhiser) [inline static/app/components/acl/featureDisabled.tsx:73]: Want to convert this into a FC while you're in here 😏

Comment (vuluongj20) [reply static/app/components/acl/featureDisabled.tsx:73]: sure thing!

File: static/app/components/acl/featureDisabled.tsx [hunk 2]
```diff
@@ -182,18 +156,32 @@ const HelpDescription = styled('div')`
...
+    grid-auto-flow: column;
+    gap: ${space(1)};
   }
 `;
 
+const CopyButton = styled(Button)`
+  margin-left: auto;
+`;
+
+const Pre = styled('pre')<{insideAlert?: boolean}>`
```
Comment (evanpurkhiser) [inline static/app/components/acl/featureDisabled.tsx:183]: Why does it need this type `insideAlert` ?

File: static/app/components/acl/featureDisabled.tsx [hunk 3]
```diff
@@ -103,74 +121,30 @@ class FeatureDisabled extends React.Component<Props, State> {
...
-    return (
-      <AlertComponent type="warning">
-        <AlertWrapper>{this.renderFeatureDisabled()}</AlertWrapper>
-      </AlertComponent>
-    );
-  }
+  const AlertComponent = typeof alert === 'boolean' ? Alert : alert;
+  return (
+    <AlertComponent type="warning" showIcon expand={renderHelp()}>
+      {message ?? t('This feature is not enabled on your Sentry installation.')}
```
Comment (evanpurkhiser) [inline static/app/components/acl/featureDisabled.tsx:134]: Then you won't have to duplicate that string twice 😄

File: static/app/components/acl/featureDisabled.tsx (hunk 2 above)
Comment (vuluongj20) [reply static/app/components/acl/featureDisabled.tsx:183]: Oh right that's from a previous commit, where the `pre` would have a different, light yellow background if it's inside an alert (and light gray if it's outside). This is no longer necessary, so I'll just remove this prop.

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**